ons-core/ons-client/src/main/java/org/apache/rocketmq/ons/api/impl/rocketmq/ProducerImpl.java

```diff
@@ -169,7 +169,13 @@ public void sendAsync(Message message, SendCallback sendCallback) {
             message.setMsgID(MessageClientIDSetter.getUniqID(msgRMQ));
         } catch (Exception e) {
             LOGGER.error(String.format("Send message async Exception, %s", message), e);
-            throw checkProducerException(message.getTopic(), message.getMsgID(), e);
+            //throw checkProducerException(message.getTopic(), message.getMsgID(), e);
```

Comment:
Try not to modify the logic here, which may cause issues with the exception handling logic of the original user.

Reply:
Done. Keeping runtime exception here is much better.